ddtrace/internal/telemetry/writer.py

```diff
@@ -70,50 +88,68 @@ def send_event(self, request):
 
     def get_headers(self, request):
         # type: (Dict) -> Dict
-        """Get all telemetry api v1 request headers"""
+        """Get all telemetry api v2 request headers"""
```

Comment:
Telemetry api v2 is not necessarily backwards compatible with v1. We have to make sure we’re sending all the new mandatory fields defined in the instrumentation telemetry spec.